sdk/lib/apisets/apisetsp.h

```diff
@@ -0,0 +1,25 @@
+#ifndef APISETSP_H
+#define APISETSP_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include "apisets.h"
+
+typedef struct _ROS_APISET
+{
+    const UNICODE_STRING Name;
+    const UNICODE_STRING Target;
+    DWORD dwOsVersions;
+} ROS_APISET;
+
+extern const ROS_APISET g_Apisets[];
+extern const LONG g_ApisetsCount;
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif APISETSP_H
```

Comment:
Should fix build break...
```suggestion
#endif // APISETSP_H
```